### PR TITLE
Add Matthias Bünger as JUnit Pioneer maintainer

### PR DESCRIPTION
**Pull request by Michael1993**
Add Matthias Bünger (Bukama) to JUnit Pioneer developers.
https://github.com/Bukama

### CODE REVIEW

File: _data/projects.yml [hunk 1]
```diff
@@ -49,6 +49,10 @@
       spoken_languages:
         - EN
         - DE
+    - name: Matthias Bünger
```
Comment (fbiville) [inline _data/projects.yml:52]: You can also add a social media URL if you want
(see [example](https://github.com/hack-commit-push/paris2021/blob/708a6576b775f8ee91866859ef80a3878c31fd71/_data/projects.yml#L80))

**Review by fbiville — COMMENTED** [review]
See question inline

File: _data/projects.yml (hunk 1 above)
Comment (Michael1993) [reply _data/projects.yml:52]: Merge conflict fixed, social added (I had to ask them).

**Comment by fbiville** [comment]
Merged as 2abe875e70560ff725e1105d2c76f5ad88df7933, thanks!

**Pull request closed without merging**